en/flight_controller/intel_aero.md

```diff
@@ -93,3 +93,16 @@ The pinout for the Lidar Lite V3 is as follows
 ![](../../assets/hardware/Aero_I2C_splitter.JPG)
 
 ![](../../assets/hardware/Aero_LidarLite.JPG)
+
+## Using Optical Flow on the Aero
+
+The Intel Aero comes with a preinstalled optical flow binary (OS version 1.6 or higher) which enables the Aero to stably fly based on vision. In order to use optical flow, a range sensor has to be installed first (see above).
+
+To use the optical flow, just run
+```
+systemctl start aero-optical-flow
+```
+on the Aero. If you want to start the optical flow binary at boot, use
+```
+systemctl enable aero-optical-flow #use disable to undo
+```
```

Comment:
Really depends on your responses above, but perhaps could re-write this similar to below:
----

## Using Optical Flow on the Aero

> **Note** In order to use optical flow, a range sensor must be installed (see above).

The Intel Aero comes with a preinstalled optical flow binary (PX4 v1.6 or higher) that enables the Aero to fly using optical flow for navigation rather than GPS.

To use the optical flow run the following command on PX4 console:
 ```
 systemctl start aero-optical-flow
 ```
If you want to start the optical flow binary at boot, use
 ```
 systemctl enable aero-optical-flow #use disable to undo
 ```